Core/src/org/sleuthkit/autopsy/communications/VisualizationPanel.java

```diff
@@ -663,15 +663,6 @@ protected void done() {
                     get();
                 } catch (InterruptedException | ExecutionException ex) {
                     logger.log(Level.WARNING, "CVT graph layout failed.", ex);
-                    String message = (lockedVertexModel.isEmpty())
-                            ? Bundle.VisualizationPanel_layoutFail_text(layout.getDisplayName())
-                            : Bundle.VisualizationPanel_layoutFailWithLockedVertices_text(layout.getDisplayName());
-
-                    Platform.runLater(()
-                            -> Notifications.create().owner(notificationsJFXPanel.getScene().getWindow())
```

Comment:
If we are not showing this warning.  I think you can get rid of the notificationsJFXPanel which was only there to be the owner of the notification popups.

Reply:
It's being used in snapshotButtonActionPerformed